docs/design/opm-tooling.md

```diff
@@ -81,13 +81,13 @@ ENTRYPOINT ["/opm"]
 CMD ["registry", "serve", "--database", "index.db"]
 ```
 
-Of note here is that we use a builder image to get the latest upstream released version of opm in order to call `opm registry serve` to host the gRPC API. If a developer or CI system would prefer to point to a different version of `opm` to serve their operator (perhaps one in a private release or a fork) then they just need to deliver their own version in a container and then use the `--binary-image` command. ex:
+In the above example, it's important to note that we use a builder image to get the latest upstream released version of opm in order to call `opm registry serve` to host the gRPC API. If a developer or CI system would prefer to point to a different version of `opm` to serve their operator (perhaps one in a private release or a fork) then they just need to deliver their own version in a container and then use the `--binary-image` command. ex:
```

Comment:
Nit: I realize this is not part of your change-set but while we're here, we could make `ex:`=>`For example:` to make it consistent with the rest of the doc.